File: 351004/Brazhalovich/Lab_1/src/main/java/org/example/newsapi/entity/News.java
package org.example.newsapi.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;
import java.time.LocalDateTime;
import java.util.HashSet;
import java.util.Set;

@Data
@EqualsAndHashCode(callSuper = true)
public class News extends BaseEntity {
    private Long userId; // FK
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
    // Many-to-Many имитация через хранение ID маркеров
    private Set<Long> markerIds = new HashSet<>();
}

File: 351003/brezgunov/distcomp/src/main/kotlin/com/distcomp/entity/News.kt
package com.distcomp.entity

import java.time.LocalDateTime

class News(
    var id: Long? = null,
    var title: String,
    var content: String,
    var created: LocalDateTime = LocalDateTime.now(),
    var modified: LocalDateTime = LocalDateTime.now(),
    var user: User? = null
)